cni/network/invoker_azure_test.go

```diff
@@ -238,8 +238,18 @@ func TestAzureIPAMInvoker_Add(t *testing.T) {
 				require.Nil(err)
 			}
 
-			fmt.Printf("want:%+v\nrest:%+v\n", tt.want, ipamAddResult.defaultInterfaceInfo.IPConfigs)
-			require.Exactly(tt.want, ipamAddResult.defaultInterfaceInfo.IPConfigs)
+			for _, ifInfo := range ipamAddResult.interfaceInfo {
+				switch string(ifInfo.NICType) {
```

Comment:
Should I bring in the cns package?

Reply:
Any reason why we didn't ? Given if CNS NIC type change or a new NIC type is added we need to update this test as well ?

Reply:
Not sure. I could not find historical context through the PRs.

Reply:
@jpayne3506 yes we should do it instead of defining same thing in multiple places